app/models/merchant.rb

```diff
@@ -57,40 +51,42 @@ def self.most_revenue_between_dates(params)
     end_date = params[:end].to_datetime + 1
 
     InvoiceItem.select("SUM(invoice_items.quantity * invoice_items.unit_price) AS revenue")
-    .joins(:invoice)
-    .joins("INNER JOIN transactions ON invoices.id = transactions.invoice_id")
-    .where(transactions: {result: "success"})
+    .joins(invoice: [:transactions])
+    .where(transactions: {result: 'success'})
     .where("invoices.created_at > '#{start_date}' AND invoices.created_at < '#{end_date}'")[0]
   end
 
 
-   def self.individual_revenue(params)
+  def self.individual_revenue(params)
     id = params[:id]
+
     InvoiceItem.select("SUM(invoice_items.quantity * invoice_items.unit_price) AS revenue")
-    .joins(:invoice)
-    .joins("INNER JOIN transactions ON invoices.id = transactions.invoice_id")
+    .joins(invoice: [:transactions])
     .where(transactions: {result: 'success'})
     .where(invoices: {merchant_id: id})[0]
   end
 
+  # def self.invoice_items_invoices_transactions_success
```

Comment:
Attempt at reducing code, but it ended up just adding lines and making it less readable